docs/en/wallet-instance-attestation.rst

```diff
@@ -83,7 +84,7 @@ Format of the Wallet Instance Attestation Request
 To obtain a Wallet Instance Attestation from the Wallet
 Provider it is necessary to send a Wallet Instance Attestation
 Request from the Wallet Instance containing the associated public key
-and a ``nonce`` previously requested to avoid replay attacks.
+and a ``nonce`` previously requested. A ``jwti`` is used to avoid replay attacks.
```

Comment:
```suggestion
, the ``nonce`` value previously requested and a ``jti`` value.
```

Reply:
@grausof considering we already have the nonce, is the jti required?
I mean that by just using the nonce the replay attacks can be avoided